pkg/cluster/k8sres.go

```diff
@@ -75,8 +75,12 @@ func (c *Cluster) statefulSetName() string {
 	return c.Name
 }
 
-func (c *Cluster) connectionPoolerName() string {
-	return c.Name + "-pooler"
+func (c *Cluster) connectionPoolerName(role PostgresRole) string {
+	name := c.Name + "-pooler"
+	if role == Replica {
+		name = name + "-repl"
```

Comment:
Since `c.Name` is a cluster name, I guess there should be both `pooler` and `repl` parts.

Reply:
I don't understand this, currently we are appending repl to **name** which is c.Name + "-pooler". Do you expect some thing else...?

Reply:
To clarify, that's just my misunderstanding, the question can be ignored.